sdp/src/lib.rs

```diff
@@ -19,8 +18,8 @@ mod parser;
 /// Errors from parsing and serializing SDP.
 #[derive(Debug, Error)]
 pub enum SdpError {
-    #[error("SDP parse: {0}")]
-    Parse(#[from] StringStreamError),
+    #[error("SDP inconsistent: {0}")]
+    ParseError(String),
```

Comment:
Why "inconsistent" when it really is a _parse error_ quite literally. I mean we could potentially split the error into parsing and then our subsequent validation.

Reply:
just missed it when moving around, fixed now